apps/crn-server/test/controllers/research-output.controller.test.ts

```diff
@@ -500,6 +500,18 @@ describe('ResearchOutputs controller', () => {
           }),
         );
       });
+
+      test('Should not throw error when no link is passed and its document type is Lab Resource', async () => {
+        const researchOutputEmptyLinkRequest = getResearchOutputCreateData();
+        researchOutputEmptyLinkRequest.documentType = 'Lab Resource';
+        researchOutputEmptyLinkRequest.link = undefined;
+
+        await researchOutputs.create(researchOutputEmptyLinkRequest);
+        expect(researchOutputDataProviderMock.fetch).not.toHaveBeenCalledWith({
+          filter: { link: expect.any(String) },
+          includeDrafts: true,
+        });
+      });
```

Comment:
You will want to test that the payload sends the `link` as `null` or `undefined` and not `''` as well.